frontend/lib/src/components/elements/Expander/Expander.tsx

```diff
@@ -136,7 +219,6 @@ const Expander: React.FC<ExpanderProps> = ({
             }),
             props: {
               className: "streamlit-expanderHeader",
-              isStale,
```

Comment:
Why remove this out of curiosity?

Reply:
It causes this error: 
<img width="782" alt="image" src="https://github.com/streamlit/streamlit/assets/2852129/44278c4e-d7be-401b-bea7-2ea5c768c847">

And it doesn't seem to change anything if I remove it. But not 100% sure why this was initially added

Reply:
I did double-check this, and it doesn't seem that anything is impacted if I leave that out.